com.unity.postprocessing/PostProcessing/Shaders/Builtins/MultiScaleVODownsample1.compute

```diff
@@ -56,9 +56,19 @@ groupshared float g_CacheW[256];
 #endif
 
 CBUFFER_START(CB0)
-    float4 ZBufferParams;
+    float4 ProjectionParams;    // x: 1 (-1 flipped), y: near,     z: far,       w: 1/far
```

Comment:
It seems that only the `y` and `z` parameters from `ProjectionParams` are used, and only the `w` parameter from `OrthoParams` - is it worth packing these into a single float4?

Reply:
Not worth it for this use case indeed, but I just made it match ProjectionParams used in fragment shaders for consistency.